radixdlt-java/src/main/java/com/radixdlt/client/application/RadixApplicationAPI.java

```diff
@@ -1099,6 +1099,10 @@ public void discoverNodes() {
 		return this.universe.getNetworkController().getActions();
 	}
 
+	public BigDecimal getMinimumRequiredFee(Atom atom) {
```

Comment:
Also here should be "atomWithoutFees".  Sorry missed this one the first time round.